cpplint_clitest.py

```diff
@@ -186,7 +186,7 @@ def get_extra_command_args(self, cwd):
         name_func=lambda fun, _, x: f"test{x.args[0].capitalize()}Sample-{x.args[1]}",
```

Comment:
```suggestion
        name_func=lambda fun, _, x: f"test_{x.args[0]}_sample-{x.args[1]}",
```
(I wonder if we should change the hyphen after "sample", by the way? Maybe we should even remove the "-sample"s from the folder names.)

Reply:
I wonder why the tests pass even when the folder name is misspelled.

Reply:
-->
* #343

Reply:
Are the folder names misspelled? They worked and responded to .def changes to me.